### PR TITLE
readme generator enhancements

### LINKED ISSUE #1755: README: Add license from package.json to generated READMEs

### PR DESCRIPTION
**Pull request by karen-stepanyan**
## Changes

- Modified readme generator to include following changes in generated readmes
Version badge instead of version string
EA base url
EA license
Shared input parameters
- Updated readme file of bootstrap folder to include information about base input parameters

## Quality Assurance

- [x] Ran `yarn changeset` if adapter source code was changed
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [ ] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [ ] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [ ] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: packages/scripts/src/generate-readme/generator.ts [hunk 1]
```diff
@@ -163,7 +175,7 @@ export class ReadmeGenerator {
       ? buildTable(tableText, paramHeaders)
       : 'There are no input parameters for this adapter.'
 
-    this.readmeText += `## Input Parameters\n\n${inputParamTable}\n\n---\n\n`
+    this.readmeText += `## Input Parameters\n\nEvery EA supports base input parameters from [this list](https://github.com/smartcontractkit/external-adapters-js/tree/develop/packages/core/bootstrap#base-input-parameters)\n\n${inputParamTable}\n\n`
```
Comment (austinborn) [inline packages/scripts/src/generate-readme/generator.ts:178]: I would make this a relative link so it works in every branch independently:
```suggestion
    this.readmeText += `## Input Parameters\n\nEvery EA supports base input parameters from [this list](../../../core/bootstrap#base-input-parameters)\n\n${inputParamTable}\n\n`
```

File: packages/scripts/src/generate-readme/generator.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ import {
 import { balance } from '@chainlink/ea-factories'
 import { getBalanceTable, inputParamHeaders, paramHeaders } from './tableAssets'
 import { EndpointDetails, EnvVars, IOMap, JsonObject, Package, Schema } from '../shared/docGenTypes'
+import { DEFAULT_WS_API_ENDPOINT } from '../../../sources/tradingeconomics/src/config'
```
Comment (austinborn) [inline packages/scripts/src/generate-readme/generator.ts:15]: This isn't needed anymore, right?

Comment (karen-stepanyan) [reply packages/scripts/src/generate-readme/generator.ts:15]: oh shoot. my IDE loves to autoimport everything :)

Comment (austinborn) [reply packages/scripts/src/generate-readme/generator.ts:15]: Lol it happens to me a lot too

**Review by austinborn — APPROVED** [review]

**Review by austinborn — APPROVED** [review]

**Pull request merged**